src/main/java/org/commcare/suite/model/RemoteQueryDatum.java

```diff
@@ -26,7 +29,7 @@ public class RemoteQueryDatum extends SessionDatum {
     private OrderedHashtable<String, QueryPrompt> userQueryPrompts;
     private boolean useCaseTemplate;
     private boolean defaultSearch;
-    private String titleLocaleId;
+    private Text titleLocaleId;
```

Comment:
Since this is a now a `Text` instead of a locale id, `titleLocaleId` doesn't make sense as a name. How about just `title`? (This goes for the various related function names, too.)